front/app/api/campaigns/useCampaign.ts

```diff
@@ -0,0 +1,21 @@
+import { useQuery } from '@tanstack/react-query';
+import { CLErrors } from 'typings';
+import fetcher from 'utils/cl-react-query/fetcher';
+import campaignsKeys from './keys';
+import { ICampaign, CampaignsKeys } from './types';
+
+const fetchCampaignExample = ({ campaignId }: { campaignId: string | null }) =>
```

Comment:
I don't think it makes sense this can be `null`, because we can't fetch a campaign without.

I see where `useCampaign` is used. I think we can be stricter on campaignId in ExampleModal and only render it when we have a campaign id.